modules/cluster-logging-log-forward-update.adoc

```diff
@@ -0,0 +1,102 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging-external.adoc
+
+// Might not be needed. See https://issues.redhat.com/browse/LOG-654
+
+[id="cluster-logging-log-forward-update_{context}"]
+= Updating log forwarding custom resources
+
+The {product-title} Log Forward API has been promoted from Technology Preview to Generally Available in {product-title} 4.6. The GA release contains some improvements and enhancements that require you to make a change to your Cluster Logging custom resource (CR) and to replace your Log Forwarding custom resource (CR) with a `ClusterLogForwarder` instance.
+
+
+.Procedure 
+
+If you use the Log Forwarding API to forward logs, after the Elasticsearch Operator and Cluster Logging Operator are fully updated to 4.6, you must make the following modifications:
+
+. Edit the Cluster Logging custom resource (CR) to remove the `logforwardingtechpreview` annotation:
++
+.Sample Cluster Logging CR
+[source,yaml]
+----
+apiVersion: "logging.openshift.io/v1"
+kind: "ClusterLogging"
+metadata:
+  annotations:
+    clusterlogging.openshift.io/logforwardingtechpreview: enabled <1>
+  name: "instance"
+  namespace: "openshift-logging"
+....
+
+----
+<1> Remove the remove the `logforwardingtechpreview` annotation.
+
+. Edit the Log Forwarding CR to create a `ClusterLogForwarder` instance:
```

Comment:
We couldn't edit the TP Forwarding CR directly. We should export it, modify locally and create a new GA LF CR
oc get LogForwarding instance -o yaml| tee ClusterLogForwarder.yaml

For example http://pastebin.test.redhat.com/909891

Drop the useless fields  like: .status, metadata.managedFields .creationTimestamp and etc.